test/tools/unit_test_runner.d

```diff
@@ -248,6 +248,7 @@ void writeCmdfile(string path, string runnerPath, string outputPath,
         "-version=GC",
         "-version=NoMain",
         "-version=MARS",
+        "-version=ExtraLocation",
```

Comment:
This needs to be added to the Dub file as well. To the lexer subpackage: https://github.com/dlang/dmd/blob/37556c0c3f7aca3e37db02631ef2e2c9ffb8a57b/dub.sdl#L16

Otherwise, when the Dub package is used as a library, it won't have this version identifier enabled.